inc/category.class.php

```diff
@@ -37,6 +37,10 @@
 
 class PluginFormcreatorCategory extends CommonTreeDropdown
 {
+   static function canView() {
+      return Session::haveRight('ticket', CREATE);
```

Comment:
rightname of a dropdown is "dropdown" (defined in CommonDropdown)

If you redefined this method this way, you may break access to the list of categories for administrators.

I think he following will be better

```suggestion
      return parent::canView() || $_SESSION['glpiactiveprofile']['interface'] == 'helpdesk';
```